website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -127,6 +133,21 @@ The `term` block the following arguments:
 - `duration` - (Optional) **DEPRECATED:** Use `threshold_duration` instead. The duration of time, in _minutes_, that the threshold must violate for in order to create a violation. Must be within 1-120 (inclusive).
 - `time_function` - (Optional) **DEPRECATED:** Use `threshold_occurrences` instead. The criteria for how many data points must be in violation for the specified threshold duration. Valid values are: `all` or `any`.
 
+## Signal
+
+The stream of telemetry data that's watched and alerted on. [Configuration of the signal](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#advanced-signal) defines what the NRQL condition will use to evaluate.
+
+For more information, see [Streaming Alerts](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/streaming-alerts-key-terms-concepts/).
+
+The `signal` block supports the following arguments:
+
+- `aggregation_window` - controls the duration of the time window used to evaluate the NRQL query, in seconds. The value must be at least 30 seconds, and no more than 15 minutes (900 seconds). Default is 60 seconds.
+- `fill_option` - determines the type of value that should be used to fill gaps (empty windows). Defaults to `STATIC`. Outlier NRQL conditions may only use `NONE`.
+- `fillValue` - used for filling if using the `STATIC` fill option. Defaults to 0.
+- `aggregation_method` - determines when we consider an aggregation window to be complete so that we can evaluate the signal for violations. Default is `CADENCE`.
```

Comment:
@akane0915 IIRC `CADENCE` is the default "for now" with `EVENT_FLOW` to become the default shortly after GA. Do we have follow-up work logged for that?

Reply:
Should also enumerate the options here like we do with `fill_option`